src/components/dataTable/demoBasicUsage/index.html

```diff
@@ -0,0 +1,44 @@
+<div class="container" layout="column" ng-controller="AppCtrl as ctrl">
+
+  <div layout="row" layout-wrap class="options">
+    <md-checkbox ng-model="ctrl.options.rowSelection">Enable Row Selection</md-checkbox>
+    <md-checkbox ng-model="ctrl.options.multiple">Multiple Row Selection</md-checkbox>
+    <md-checkbox ng-model="ctrl.options.auto">Select on Row Click</md-checkbox>
+  </div>
+
+  <md-divider></md-divider>
+
+  <md-content>
+
+    <table md-table md-row-select="{{ctrl.options.rowSelection}}" md-selected="ctrl.selected" multiple="{{ctrl.options.multiple}}">
+      <thead md-head md-order="ctrl.order" md-on-reorder="$log.info(ctrl.order)">
+        <tr md-row>
+          <th md-cell md-order-by="name">Dessert (100g serving)</th>
+          <th md-cell md-align="right" md-order-by="calories">Calories</th>
+          <th md-cell md-align="right" md-order-by="fat">Fat (g)</th>
+          <th md-cell md-align="right" md-order-by="carbs">Carbs (g)</th>
+          <th md-cell md-align="right" md-order-by="protein">Protein (g)</th>
+          <th md-cell md-align="right" md-order-by="sodium">Sodium (mg)</th>
+          <th md-cell md-align="right" md-order-by="calcium">Calcium (%)</th>
+          <th md-cell md-align="right" md-order-by="iron">Iron (%)</th>
```

Comment:
We should reduce some rows here, because the demo looks a bit weird with having scrollbars.